docs/breez-sdk/src/guide/receive_lnurl_pay.md

```diff
@@ -0,0 +1,306 @@
+<h1 id="lightning-address">
+    <a class="header" href="#lightning-address">Using Lightning Addresses</a>
+</h1>
+
+<h2 id="what-is-lightning-address">
+    <a class="header" href="#what-is-lightning-address">What is a Lightning Address?</a>
+</h2>
+
+A Lightning Address is a human-readable identifier formatted like an email address (e.g., `user@domain.com`) that can be used to receive Bitcoin payments over the Lightning Network. Behind the scenes, it uses the LNURL-pay protocol to dynamically generate invoices when someone wants to send a payment to this address.
+
+<h2 id="lnurl-server">
+    <a class="header" href="#lnurl-server">lnurl server</a>
+</h2>
+
+To use Lightning Addresses with the Breez SDK, you first need to supply a domain. There are two options:
+
+1. **Use Breez's LNURL server**: You can have your domain added to the LNURL server run by Breez.
+2. **Self-hosted LNURL server**: You can run your own LNURL server.
+
+<h2 id="configuring-lightning-address">
+    <a class="header" href="#configuring-lightning-address">configuring lightning address</a>
+</h2>
+To configure the domain in the sdk, you need to pass the `lnurl_domain` parameter in the SDK configuration:
+
+<custom-tabs category="lang">
+<div slot="title">Rust</div>
+<section>
+
+```rust,ignore
+let config = Config {
+    api_key: Some("your-api-key".to_string()),
+    network: Network::Bitcoin,
+    lnurl_domain: Some("yourdomain.com".to_string()),
+    // ... other config options
+};
+```
+</section>
+
+<div slot="title">Swift</div>
+<section>
+
+```swift,ignore
+let config = Config(
+    apiKey: "your-api-key",
+    network: Network.bitcoin,
+    lnurlDomain: "yourdomain.com",
+    // ... other config options
+)
+```
+</section>
+
+<div slot="title">Kotlin</div>
+<section>
+
+```kotlin,ignore
+val config = Config(
+    apiKey = "your-api-key",
+    network = Network.BITCOIN,
+    lnurlDomain = "yourdomain.com",
+    // ... other config options
+)
+```
+</section>
+
+<div slot="title">Dart</div>
```

Comment:
For consistency and we don't directly support base Dart. Same for snippets below 
```suggestion
<div slot="title">Flutter</div>
```